pkg/e2e/operators/splunkforwarder.go

```diff
@@ -21,8 +21,8 @@ var splunkForwarderBlocking string = "[Suite: operators] [OSD] Splunk Forwarder
 var splunkForwarderInforming string = "[Suite: informing] [OSD] Splunk Forwarder Operator"
 
 func init() {
-	alert.RegisterGinkgoAlert(splunkForwarderBlocking, "SD-SREP", "Matt Bargenquast", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
-	alert.RegisterGinkgoAlert(splunkForwarderInforming, "SD-SREP", "Matt Bargenquast", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
+	alert.RegisterGinkgoAlert(splunkForwarderBlocking, "SD-SREP", "Matt Bargenquast", "@srep-security-team", "sd-cicd@redhat.com", 4)
```

Comment:
@dustman9000 this updates the slack channel, rather then the Owner. Please see https://github.com/openshift/osde2e/pull/952/files for reference.